### DISCUSSION TITLE
Terminal's default colorschema

### DISCUSSION THREAD

**Discussion by batyrmuhadov** [NvChad/NvChad#2768] [Q&A] [answered]
Is there any option to disable nvchad's colorschemas and use default terminal??

**Comment by siduck** [comment]
@batyrmuhadov no, why would you want that?

**Reply by batyrmuhadov** [reply]
because I like my it default more than nvchad's and don't want overwrite configs

**Reply by siduck** [reply]
you cant change the syntax stuff, but can make the nvim bg same as terminal bg

**Reply by batyrmuhadov** [reply]
Could you guide me how to do it?

**Reply by siduck** [reply, marked as answer]
@batyrmuhadov either set transparency in chadrc or  set Normal highlight group's bg to none https://nvchad.com/docs/config/theming/

**Reply by batyrmuhadov** [reply]
yes makes sense, thank you!
